scripts/release/verify-fdroid-native-libs.sh

```
@@ -14,16 +15,149 @@ require_cmd() {
+    found="$(find "$sdk_root/ndk" -type f -path '*/toolchains/llvm/prebuilt/*/bin/llvm-readelf' | sort | tail -n 1)"
+  fi
+  if [[ -n "$found" ]]; then
+    printf '%s\n' "$found"
+    return
+  fi
+  command -v readelf || command -v llvm-readelf
+}
+
+property_value() {
+  local name="$1"
+  grep -E "^[[:space:]]*$name[[:space:]]*=" "$KEY_PROPERTIES" |
```

Shellcheck (0.11.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)